corral/cmd/cmd_update.pony

```diff
@@ -74,7 +73,8 @@ actor _Updater
       ctx.log.fine("")
 
       // TODO: consider parsing version much earlier, before we even get here.
-      // TODO: also consider allowing literal tags and not just constraints expressions.
+      // TODO: also consider allowing literal tags and not just constraints
+      //   expressions.
```

Comment:
@Theodus we have nothing in the style-guide about multiline comments like this.

for a TODO: that is multiline is feels like "expressions" should line up under "also".

That said that is starting to be a very specific rule that would suck to automate.
I think it should be that "expressions" lines up with TODO.

Thoughts?

Reply:
I personally tend to keep the spacing consistent on all comment lines (so the 'e' in "expressions" would lineup with the 'T' in TODO). I'm not sure that my opinion on this one is what we should go with or if this should even be in the style guide.

Reply:
@Theodus you wrote the style guide. i defer to you on any questions of ambiguity. as far as I am concerned. you are the arbiter at this point.

Reply:
Ok, then what I said is probably the way to go since it's consistent with the docstring examples in the guide.

Reply:
Sounds good. Carl can you make that change?

@Theodus when you have time, can you update the style guide to address the ambiguities i pointed out here?

and boy o boy, i can't wait til we have a ponyfmt.